crm_helpdesk/README.rst

```diff
@@ -0,0 +1,23 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+============
+Crm Helpdesk
+============
+
+Like records and processing of claims, Helpdesk and Support are good tools
+to trace your interventions. This menu is more adapted to oral communication,
+which is not necessarily related to a claim. Select a customer, add notes
+and categorize your interventions with a channel and a priority level.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Miquel Raïch <miquel.raich@eficent.com>
```

Comment:
This is for Aleph. If finally is put on OCA, then I we should put that thing you said